task3.js

```diff
@@ -0,0 +1,18 @@
+// Write a function that merges two arrays, removing duplicate elements.
+
+function mergeArrays(arr1, arr2) {
```

Comment:
it ok, but i would recomend u to add some check on array
 i mean Array.isArray(arr1) e.t.c